docs/content/em/api_NSEM.rst

```diff
@@ -0,0 +1,54 @@
+.. _api_NSEM:
+
+Natural Source Electromagnetics
+*******************************
+
+.. todo:: Docs here
+
+SimPEG (Simulation and Parameter Estimation in Geophysics) is a python
+package for simulation and gradient based parameter estimation in the
+context of geoscience applications.
+
+Uses SimPEG as the framework for the forward and inverse
```

Comment:
In this docstring, could you list the types of problems implemented (doesn't need to be much right now, just enough for people to make the mapping). Eg. something along the lines of 
```
The NSEM package within SimPEG currently includes
- Magnetotellurics 
- Tipper
future development will include
- ZTEM
```